### PR TITLE
Switch to checking for CrashLoopBackOff to show the container looping message

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://github.com/openshift/origin/issues/5549
Fixes https://github.com/openshift/origin/issues/2815

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett review

**Comment by spadgett** [comment]
Also fixes #2815

File: assets/app/scripts/filters/resources.js [hunk 1]
```diff
@@ -458,7 +448,7 @@ angular.module('openshiftConsole')
             warnings.push({reason: "Failed", message: "The container " + containerStatus.name + " failed with a non-zero exit code " + containerStatus.state.terminated.exitCode + "."});
           }
           if (isContainerLoopingFilter(containerStatus)) {
-            warnings.push({reason: "Looping", message: "The container " + containerStatus.name + " is restarting frequently, which usually indicates a problem. It has restarted " + containerStatus.restartCount + " times, and has restarted within the last five minutes."});
+            warnings.push({reason: "Looping", message: "The container " + containerStatus.name + " is crashing frequently. It must wait before it will be restarted again."});
```
Comment (spadgett) [inline assets/app/scripts/filters/resources.js]: Nit: Is it always "crashing frequently" or is one crash enough for the status?

**Comment by spadgett** [comment]
LGTM

File: assets/app/scripts/filters/resources.js (hunk 1 above)
Comment (jwforres) [inline assets/app/scripts/filters/resources.js]: checked with clayton, more than one crash is needed to get into this status

**Comment by jwforres** [comment]
@openshift/team-project-committers

**Comment by smarterclayton** [comment]
Approved on Jessica's ack.

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
[test]

**Comment by spadgett** [comment]
[test]

**Comment by jwforres** [comment]
[test]

**Pull request merged**